packages/manager/src/managers/SliceMachineManager.ts

```diff
@@ -3,10 +3,7 @@ import {
 	CustomType,
 } from "@prismicio/types-internal/lib/customtypes";
 import { SharedSliceContent } from "@prismicio/types-internal/lib/content";
-import {
-	ForbiddenError,
-	UnauthorizedError,
-} from "@prismicio/custom-types-client";
+import * as prismicCustomTypesClient from "@prismicio/custom-types-client";
```

Comment:
We can import like before, but I changed it to a namespace import because:

1. It is consistent with other imports of `@prismicio/custom-types-client` in `@slicemachine/manager`.
2. It makes it clear that `ForbiddenError` and `UnauthorizedError` come from `@prismicio/custom-types-client` (e.g. `prismicCustomTypesClient.ForbiddenError`).

   I felt this was important because we have a Slice Machine-specific `UnauthorizedError` in `src/errors.ts`, which could cause confusion with `instanceof` checks.